doc/user/content/manage/dbt/_index.md

```diff
@@ -451,9 +463,14 @@ unspecified, the default database for the connection is used.
 
 #### Indexes
 
-Use the `indexes` option to define a list of [indexes](/sql/create-index/) on
-`source`, `view`, `table` or `materialized view` materializations. Each index
-option can have the following components:
+Use the `indexes` option to define a list of [indexes](/concepts/indexes/) on
+`source`, `view`, `table` or `materialized view` materializations. In
+Materialize, [indexes](/concepts/indexes/) on a view maintain view results in
+memory within a [cluster](/concepts/clusters/ "pools of compute resources (CPU,
+memory, and scratch disk space)"). As the underlying data changes, indexes
+**incrementally update** the view results in memory.
```

Comment:
Once https://github.com/MaterializeInc/materialize/pull/29090 is merged, will also sprinkle some links to the usage patterns section in this page.

Reply:
Can we make the changes to the dbt documentation in a separate PR?